readthedocs/templates/homepage.html

```diff
@@ -133,23 +133,39 @@ <h3>{% trans "Featured Projects" %}</h3>
   {% endif %}
   {% endcache %}
 
+  <section>
+    <h2>{% trans 'About Read the Docs' %}</h2>
+
+    <p>
+      Read the Docs has grown substantially
+      since its beginning as a weekend project and is closing in on being a top-1000 site on the internet.
+      Today, we:
+    </p>
+
+    <!-- Updated: June 2018, pageviews June 2021 -->
+    <ul class="donate-about">
+      <li>Serve over <b>55 million pages</b> of documentation a month</li>
+      <li>Serve over <b>40 TB</b> of documentation a month</li>
+      <li>Host over <b>80,000 open source projects</b> and support over <b>100,000 users</b></li>
+      <li>Are supported by a <a href="https://docs.readthedocs.io/en/latest/team.html">very small team</a> of dedicated engineers</li>
+    </ul>
+
+  </section>
   <!-- Funding and Contributing -->
   <section>
     <h2>{% trans "Read the Docs is funded by the community" %}</h2>
     <p>
 
-    {% url "donate" as sponsors_url %}
-    {% url "gold_detail" as gold_detail %}
+    {% url "donate" as donate_url %}
 
     {% blocktrans trimmed %}
-    Read the Docs is a huge resource that millions of developers rely on
-    for software documentation. It would not be possible without the
-    support of our
-    <a href="{{ sponsors_url }}">sponsors</a>,
-    <a href="https://www.ethicalads.io/advertisers/?ref=rtd">advertisers</a>,
-    and <a href="{{ gold_detail }}">readers like you</a>.
-    {% endblocktrans %}
+    <p>
+      We fund our operations through <a href="{{ donate_url }}">donations</a>,
```

Comment:
Small nitpick here, but the ordering plays down the importance of our commercial hosting. Our donations are only a small amount of monthly income. I think my issue with this is mostly listing donations first.

Reply:
Absolutely, will change it.